package.json

```diff
@@ -4,13 +4,13 @@
   },
   "bundleDependencies": false,
   "dependencies": {
-    "@adobe/aemcs-api-client-lib": "git+https://github.com/adobe/aemcs-api-client-lib.git#main",
-    "cross-fetch": "^3.0.6"
+    "@adobe/aemcs-api-client-lib": "git+https://github.com/adobe/aemcs-api-client-lib.git#main"
   },
   "deprecated": false,
   "description": "AEM Headless SDK Client",
   "devDependencies": {
     "@adobe/eslint-config-aio-lib-config": "^1.2.1",
+    "cross-fetch": "^3.1.4",
```

Comment:
The PR description mentions `Removed cross-fetch plugin`, but this looks like an addition.

Reply:
Removed from the SDK dependencies, used as devDependency for NodeJS tests.
https://github.com/adobe/aem-headless-client-js/pull/20/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L6-L8